routes/inertia.php

```diff
@@ -64,4 +74,13 @@
                         ->name('team-invitations.destroy');
         }
     });
+
+    // Socialite...
+    if (Jetstream::hasSocialiteFeatures()) {
+        Route::get('/oauth/{provider}', [SocialiteController::class, 'redirectToProvider'])->name('oauth.redirect');
+        Route::get('/oauth/{provider}/callback', [SocialiteController::class, 'handleProviderCallback'])->name('oauth.callback');
+
+        // Set user password
+        Route::put('/user/set-password', [PasswordController::class, 'store'])->name('user-password.set');
```

Comment:
Maybe this could turn into a more general concept.

```
Route::get('complete-profile', [CompleteProfileController::class, 'show'])->name('complete-profile.set');
```
```
Route::put('complete-profile', [CompleteProfileController::class, 'update'])->name('complete-profile.set');
```
Maybe session flags (missing_email, missing_password) set during auth flow determine fields shown.

Maybe a middleware to redirect back to this route if incomplete